entity/templates/src/main/java/package/service/_EntityService.java

```diff
@@ -0,0 +1,56 @@
+package <%=packageName%>.service;
+<%  var instanceType = (dto == 'mapstruct') ? entityClass + 'DTO' : entityClass;
+    var instanceName = (dto == 'mapstruct') ? entityInstance + 'DTO' : entityInstance; %>
+import <%=packageName%>.domain.<%= entityClass %>;<% if (dto == 'mapstruct') { %>
+import <%=packageName%>.web.rest.dto.<%= entityClass %>DTO;<% } if (pagination != 'no') { %>
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;<% } %>
+<% if (dto == 'mapstruct') { %>
+import java.util.LinkedList;<% } %>
+import java.util.List;
+
+/**
+ * Service Interface for managing <%= entityClass %>.
+ */
+public interface <%= entityClass %>Service {
```

Comment:
The service layer should be made @Transational -> can you copy this from the service sub-generator? It's important as this is how you handle lazy-loading errors with Hibernate, and so a good reason to have this service layer